### PR TITLE
Updated styling for studio.code.org.

### PR DESCRIPTION
**Pull request by breville**
**DONE:**
- all miniblock css in now courseblock in course-blocks.scss
- tested in Chrome/FF/Safari
- looks ok in other languages except the artist course in RTL (noted as a bug for later)

**TODO:**
- eye tests for blocks following 4 pages
- eye tests for user_hero on code.org and studio.code.org for both teacher and student
- box border only on the sides, not on the top or bottom (except when no action, then bottom border)

Code Studio Homepage (not signed in):
image

Sign in page:
image

Teacher dashboard:
image

Code Studio Homepage (signed in):
image

### CODE REVIEW

**Comment by tanyaparker** [comment]
I think HoC is also supposed to show up in sign in page. Fixing now.

File: dashboard/app/views/shared/_course_tall_block.haml [hunk 1]
```diff
@@ -35,13 +37,14 @@
...
 - if %w(course1 course2 course3 course4).include?(id)
   - audience = data_t_suffix('script.name', id, 'description_audience')
-  - block_class = 'courseblock-tall courseblock-tall-audiencerow'
-- else
-  - block_class = 'courseblock-tall'
+  
+- block_class = 'courseblock-tall'
+
+- if ! (current_user && showState)
+  - block_class << 'courseblock-noaction'
```
Comment (breville) [inline dashboard/app/views/shared/_course_tall_block.haml]: did this work?  i think it needed a space inside the single quotes like this: ' courseblock-noaction'

File: dashboard/app/views/shared/_course_wide_block.haml [hunk 2]
```diff
@@ -1,4 +1,6 @@
--if id == 'hoc'
+- showState ||= false
```
Comment (breville) [inline dashboard/app/views/shared/_course_wide_block.haml]: this means showState is an optional parameter when including the partial, but doesn't hurt to specify it explicitly everywhere

**Comment by breville** [comment]
Nice work.  Couple other things that changed and should be tested:

- user_hero panel looks good on studio.code.org for both signed in teacher and signed in student.
- user_hero panel looks good on code.org for same two cases.

File: dashboard/app/views/shared/_course_wide_block.haml (hunk 2 above)
Comment (tanyaparker) [inline dashboard/app/views/shared/_course_wide_block.haml]: At least everywhere I know we include block partials, I've specified either true/false. They're in all_courses, hoc_courses, k5_courses, and dashboard/app/views/devise/sessions/new.html.haml

**Pull request merged**